korgan/auto_payment_runtime.py

```
@@ -0,0 +1,182 @@
+            chat_id=user_id,
+            from_chat_id=admin_id,
+            message_id=admin_doc_message_id,
+            caption=_document_caption_with_review(kind, language),
+            reply_markup=_document_review_markup(kind, language),
+        )
+    except Exception:
+        LOGGER.exception("AUTO_PAYMENT_DOCUMENT_RELEASE_FAILED user=%s doc_msg=%s", user_id, admin_doc_message_id)
+        await message.answer(
+            "Оплата прошла AI-проверку, но Word-файл не удалось выдать из технического хранилища. Обратитесь в техподдержку; повторно оплачивать не нужно."
+            if language != "kk"
+            else "Төлем AI-тексеруден өтті, бірақ Word-файлды техникалық қоймадан беру мүмкін болмады. Қолдауға жазыңыз; қайта төлеудің қажеті жоқ."
```

Ruff (0.16.1)

[warning] String contains ambiguous `б` (CYRILLIC SMALL LETTER BE). Did you mean `6` (DIGIT SIX)?

(RUF001)